lib/AST/NameLookup.cpp

```diff
@@ -2690,3 +2694,197 @@ swift::getDirectlyInheritedNominalTypeDecls(
 
   return result;
 }
+
+namespace swift {
+namespace namelookup {
```

Comment:
Nitpick: You don't need these in this file; just qualify things with `namelookup::`. Or even adding a `using namespace swift::namelookup` or `using namelookup::FindLocalVal` at the top.

Reply:
Good point, done.